staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go

```diff
@@ -35,7 +35,7 @@ import (
 	"k8s.io/client-go/restmapper"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/util/homedir"
-	utilpointer "k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
```

Comment:
```
2025/07/06 21:39:46 - validating imports for 1 packages
2025/07/06 21:39:46 -- found forbidden imports for k8s.io/cli-runtime/pkg/genericclioptions:
2025/07/06 21:39:46 --- k8s.io/utils/ptr
2025/07/06 21:39:46 - FAIL
```

+++ exit code: 1
+++ error: 1
[0;31mFAILED   verify-imports.sh	51s

/hold

Reply:
Mhhh.. wondering, what the problem is. It's also the 100th change to the new package 😅